reference/tax_provider.yml

```diff
@@ -614,113 +614,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/request-adjust'
-              examples:
```

Comment:
I removed the incorrect example, but if there's a good example we could use, we could add it in

Reply:
It’d be the same example as any of the others, although we could get fancy with it I suppose. I think it’s probably fine to simplify.